tests/tensorflow/test_sanity_sample.py

```diff
@@ -362,3 +364,28 @@ def test_export_with_resume(_config, tmp_path, export_format, _case_common_dirs)
     model_path = os.path.join(export_path, 'saved_model.pb') \
         if export_format == 'saved-model' else export_path
     assert os.path.exists(model_path)
+
+
+def get_prepare_checkpoint_configs():
+    supported_model_types = ['object_detection', 'segmentation']
+    configs = []
+    for model_type, config_paths in CONFIGS.items():
+        if model_type in supported_model_types:
+            for config_path in config_paths:
+                configs.append((model_type, config_path))
+    return configs
+
+
+@pytest.mark.parametrize('model_type,config_path', get_prepare_checkpoint_configs(),
+                         ids=[x[0] for x in get_prepare_checkpoint_configs()])
+def test_prepare_checkpoint(model_type, config_path):
+    dirpath = tempfile.mkdtemp()
+    args = {
+        '--model-type': model_type,
+        '--config': config_path,
+        '--checkpoint-save-dir': dirpath,
+        '--resume': dirpath,
+    }
+
+    prepare_checkpoint_main(convert_to_argv(args))
```

Comment:
Maybe it worth adding a test that loads the saved model, which makes sure that the model is saved in a proper way. What do you think?

Reply:
@daniil-lyakhov  please, estimate the impact on pre-commit time.

Reply:
Ok, I'll add test and check running time

Reply:
I've added evaluation for prepare checkpoint. test_prepare_checkpoint took ~48 second to execute it locally